warehouse/query-core/src/test/java/datawave/query/index/lookup/IntersectionTest.java

```diff
@@ -209,4 +214,458 @@ public void testIntersection_EmptyExceededValueThreshold() throws ParseException
         Intersection intersection = new Intersection(iterable, null);
         assertEquals(IndexStream.StreamContext.ABSENT, intersection.context());
     }
+    
+    private ScannerStream buildScannerStream(String shard, String field, String value, List<String> uids) {
+        IndexInfo ii1;
+        if (uids != null) {
+            ii1 = new IndexInfo(uids);
+        } else {
+            ii1 = new IndexInfo(-1);
+        }
+        JexlNode n1 = JexlNodeFactory.buildEQNode(field, value);
+        ii1.applyNode(n1);
+        Iterator<Tuple2<String,IndexInfo>> i1 = Arrays.asList(new Tuple2<>(shard, ii1)).iterator();
+        return ScannerStream.variable(i1, n1);
```

Comment:
Won't hold up an approval, but I would rather just see the variable names "indexInfo" and "node"